config/locales/diaspora/en.yml

```diff
@@ -1091,18 +1091,23 @@ en:
 
   registrations:
     new:
+      title: "Join the community!"
```

Comment:
I think it's worth specifying: "Join the diaspora* community!"

Reply:
OK.